pallets/loans/src/tests.rs

```diff
@@ -556,6 +572,20 @@ fn ratio_and_rate_works() {
     })
 }
 
+#[test]
+fn set_liquidation_incentive_updates_stored_values() {
+    ExtBuilder::default().build().execute_with(|| {
+        let _ = Loans::set_liquidation_incentive(Origin::root(), DOT, Ratio::one());
+        let _ = Loans::set_liquidation_incentive(Origin::root(), NATIVE, Ratio::one());
+
+        assert_eq!(
+            LiquidationIncentive::<Runtime>::try_get(DOT).unwrap(),
+            Ratio::one()
+        );
+        assert!(LiquidationIncentive::<Runtime>::try_get(NATIVE).is_err());
```

Comment:
Why is error for native token and ok for DOT?

Reply:
Native token does not have a genesis liquidation -> https://github.com/parallel-finance/parallel/blob/ee12e6e76e32ce36ecca6c76d57993fb92e9a081/pallets/loans/src/mock.rs#L246-L251